app/composition/workflows/whatsapp-llm-failure-retry/workflow-steps.ts

```
@@ -0,0 +1,150 @@
+	await saveOutboundWebhookLog({
+		webhookLogRepository: dependencies.webhookLogRepository,
+		requestUrl: params.requestUrl,
+		fromNumber: params.payload.toNumber,
+		toNumber: params.payload.fromNumber,
+		response: aiResponse,
+		messageSid: sendResult?.sid,
+		status: sendResult ? "sent" : "not_sent",
+	});
+
+	return { status: sendResult ? "sent" : "send_failed" };
+}
```

Biome (2.4.4)

[error] This function has too many lines (79). Maximum allowed is 60.

(lint/complexity/noExcessiveLinesPerFunction)